src/einsteinpy/metric/kerr_null.py

```diff
@@ -0,0 +1,83 @@
+import warnings
+
+import astropy.units as u
+import numpy as np
+
+from einsteinpy import constant
+from einsteinpy.integrators import RK45
+from einsteinpy.utils import schwarzschild_radius as scr
+
+# paper : https://arxiv.org/pdf/1601.02063.pdf
+# paper : http://www.math.mcgill.ca/gantumur/math599w17/project-kerr.pdf
+
+_c = constant.c.value
+_G = constant.G.value
+
+
+class KerrNull:
+    """
+    Class for defining methods for Kerr Null Geodesics (Schwarzschild Null Geodesics when J=0)
```

Comment:
My proposal, how about we define a class explicitly, which inherits this class. Which is Schwarzschild Null?

Reply:
Yes. It's a good idea.
But let's first write KerrNull